notebooks/signac-flow_Aggregate_Tutorial.ipynb

```diff
@@ -0,0 +1,219 @@
+{
```

Comment:
<p>Since this is a tutorial, it should explain a little about how <code>make_aggregates</code> works. What is <code>aggregates</code>, and how is it generated from the 2 arguments that got passed to <code>make_aggregates</code>? </p><p> </p><p>Edit: I guess what I'm getting at is that it should be easy for someone to look at this and know how to apply it to their own project.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Yes, that's true. I'll add the explaination